docs/tsc/meetings/2019-08-05.md

```diff
@@ -0,0 +1,85 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+August 5, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Michael Dolan
+
+Attendees:
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+
+Apologies:
+  Mark Boorer
+
+# **OCIO TSC Meeting Notes**
+
+* License Scan Status
+    - Michael: Test LUT files were called out as not having license notices. Can 
+      we have an exception with these?
+    - John: The ASWF governing board can approve exceptions. They will need to 
+      approve the sampleicc exception.
+    - Doug: Could put a license file in the test data directory which applies 
+      to all files.
+    - John: Agrees that would be a good approach. BSD-3-Clause is the most 
+      appropriate license for these files.
+    - **TODO**: Michael will add LICENSE.md file to test/data dir in PR #783
+    - **TODO**: Larry will discuss the flagged “Cloudy with a Chance of 
+      Meatballs” images with Sony.
+    - Kevin: We also need to update the LICENSE on the OCIO website
+    - Michael: That is pulled from GitHub, but currently from the RB-1.1 
+      branch. since the license cleanup work is being done in master, is it Ok 
+      to wait for a 2.0 release next year to update the site?
+    - John: That should be Ok.
+    - Kevin: Let’s make a note to not forget it.
```

Comment:
Should this be an Issue?

Reply:
#790 - more of a general reminder, but inherently accounts for the License page.